src/cleanlab_tlm/utils/chat.py

```diff
@@ -6,10 +6,14 @@
 
 from __future__ import annotations
```

Comment:
Could you add some unit tests to this PR?  That will greatly help reviewers, if they show what the expected string representation is of the various messages-reformatting that happen in the new code.

See:  https://github.com/cleanlab/cleanlab-tlm/blob/main/tests/test_chat.py